src/styles/utils/_variables.scss

```
@@ -0,0 +1,18 @@
+$colorMain: #060b35;
```

use kebab-case for variable naming
```suggestion
$color-main: #060b35;
```